src/Phinx/Migration/Manager.php

```diff
@@ -368,61 +391,89 @@ public function executeSeed($name, SeedInterface $seed)
      * Rollback an environment to the specified version.
      *
      * @param string $environment Environment
-     * @param int $version
+     * @param int $target
      * @param bool $force
+     * @param bool $targetMustMatchVersionName
      * @return void
      */
-    public function rollback($environment, $version = null, $force = false)
+    public function rollback($environment, $target = null, $force = false, $targetMustMatchVersionName = true)
```

Comment:
what I mean was, I used `*versionName` for several variable names, which should also be fixed, but it can lead to confusion with the fact that `version` is used for the version array-data. In this case, I guess I'll rename the variables to `*versionCreationTime`